go.mod

```diff
@@ -8,6 +8,9 @@ replace (
 	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20200121204235-bf4fb3bd569c
 )
 
+// TODO: Remove this when https://github.com/vishvananda/netlink/pull/554 is merged
+replace github.com/vishvananda/netlink => github.com/twelho/netlink v1.1.1-0.20200709104403-94a3428ca41d
```

Comment:
Maybe you could tag your repo to sth like v1.1.2-rc.1?

Reply:
I can do that, I'll just need come up with a tag that doesn't hinder upstream's versioning conventions.